src/Hazelcast.Net/Models/HOffsetDateTime.cs

```diff
@@ -82,8 +83,8 @@ public override string ToString()
         {
             return Offset switch
             {
-                var s when s < TimeSpan.Zero => LocalDateTime + "-" + Offset.ToString(@"hh\:mm"),
-                var s when s > TimeSpan.Zero => LocalDateTime + "+" + Offset.ToString(@"hh\:mm"),
+                var s when s < TimeSpan.Zero => LocalDateTime + "-" + Offset.ToString(@"hh\:mm", CultureInfo.InvariantCulture),
```

Comment:
Do you think `CultureInfo.InvariantCulture` is needed? We specify quite a strict format here.

Reply:
Not "needed" but this does not hurt and it gets rid of a warning.